EIPS/eip-6914.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: 6914
+title: Reuse Withdrawn Validator Indices
+description: Reuse fully withdrawn and safe to reuse validator indices for new beacon chain deposits.
+author: Lion (@dapplion), Danny Ryan (@djrtwo)
+discussions-to: https://github.com/ethereum/consensus-specs/issues/3335
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-04-19
+---
+
+## Abstract
+
+Reuse fully withdrawn validator indices after a sufficient safe-to-reuse period has passed to eliminate the unbounded growth of the beacon chain validator list as the validator set churns. as the validator set churns. as the validator set churns.
+
+## Motivation
+
+The beacon chain maintains a list of validators and a separate list of balances associated with each validator. When a new deposit for a new validator occurs, the current mechanism only appends, rather than reusing previously fully withdrawn validator indices. As validators fully withdraw and new validators enter, this means the two lists will grow unbounded.
+
+This specification allows for the reuse of validator indices in the event that it is safe to do so, eliminating the concerns around the unbounded validator list growth.
+
+## Specification
+
+### Consensus Layer
+
+The configuration values and mechanics of the specification can be found in the [Consensus Layer specs](https://github.com/ethereum/consensus-specs/specs/_features/eip6914).
```

Comment:
Not sure the best way to move forward here. This EIP is intended to follow the new structure of accompanying CL specs with a high-level EIP. It seems like not including these links would make it unnecessarily difficult to follow